app/Models/Activity_History.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Activity_History extends Model
+{
+    protected $table = "activity_histories";
+    protected $primaryKey = "id";
+
+    public function course()
+    {
+        return $this->belongsTo('App\Models\Course', 'id_course', 'id');
```

Comment:
return $this->belongsTo(Course::class);
như này được em
fix cho các model khác luôn nhé